apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -1063,6 +1064,26 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
                             </div>
                         )
                     }
+                    <div className="pt-0 mt-0">
+                        <Field
+                            data-testid={ `${ testId }-fapi-app-checkbox` }
```

Comment:
for new components, use `data-componentid`

Reply:
Fixed with [757e384](https://github.com/wso2/identity-apps/pull/4492/commits/757e384b809fa1c78dc5e6190d257fc51363d915)